policyengine_us/variables/input/income/emp_self_emp_ratio.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class emp_self_emp_ratio(Variable):
+    value_type = float
+    entity = Person
+    label = "employment-to-self-employment income ratio"
```

Comment:
While we're here, can we make this variable correctly reflect its name? Or we can file an issue for it.
```suggestion
    label = "Share of earnings from wages and salaries"
```